lib/parse-statements.js

```diff
@@ -6,20 +6,6 @@ const valueParser = require("postcss-value-parser")
 // extended tooling
 const { stringify } = valueParser
 
-function split(params, start) {
-  const list = []
-  const last = params.reduce((item, node, index) => {
-    if (index < start) return ""
-    if (node.type === "div" && node.value === ",") {
-      list.push(item)
-      return ""
-    }
-    return item + stringify(node)
-  }, "")
-  list.push(last)
-  return list
-}
```

Comment:
I overlooked that this was still present and JavaScript automatically joins arrays of strings with `,`.

This entire section has become redundant with the refactors for `v16`.